ClosedXML/Excel/RichText/XLFormattedText.cs

```diff
@@ -176,14 +225,17 @@ public bool Equals(IXLFormattedText<T> other)
             return _phonetics == null || Phonetics.Equals(other.Phonetics);
         }
 
-        public String Text { get { return ToString(); } }
+        public String Text
+        { get { return ToString(); } }
 
         private IXLPhonetics _phonetics;
-        public IXLPhonetics Phonetics 
+
+        public IXLPhonetics Phonetics
         {
             get { return _phonetics ?? (_phonetics = new XLPhonetics(_defaultFont)); }
         }
 
-        public Boolean HasPhonetics { get { return _phonetics != null; } }
+        public Boolean HasPhonetics
```

Comment:
could you as a next step make a PR for fixing all formatting issues  in one step?
this is quite hard to read doing all fixes where files are touched.